packages/template-retail-react-app/app/pages/cart/index.test.js

```diff
@@ -292,16 +285,30 @@ describe('Remove item from cart', function () {
             })
         )
     })
-    test('Can remove item from the cart', async () => {
+    test.skip('Can remove item from the cart', async () => {
```

Comment:
As per Brian's comment, I decided to skip this test: https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1100#discussion_r1162130986